functions/sttChecker/index.js

```diff
@@ -157,30 +162,30 @@ const updateTranscriptsStatus = async (
           update.groupedc = zlib.gzipSync(JSON.stringify(grouped));
           update.status = "done";
           update.runtime = getRuntime(execTimestamp, created);
-          console.log(
-            `Finished job ${jobId} in ${update.runtime.humanReadable}`
+          info(
+            `Finished job ${jobId} in ${update.runtime.humanReadable}: `, jobData
           );
         }
       }
       await updateTranscription(admin, job.id, projectId, update);
-      console.log(`Updated ${job.id} with data`, update);
+      info(`Updated ${job.id} with data ${update}`, jobData);
     } catch (err) {
-      console.error(
-        `[ERROR] Failed to get STT jobs status for ${fileName}: ${err}`
+      error(
+        `[ERROR] Failed to get STT jobs status for ${fileName}: `, { ...jobData, err}
```

Comment:
`filename` is logging with two backslashes (eg users//audio/FvI2H9m23lWHlZbEC0k9.wav), is that correct? Does this affect the call to getJobStatus on L151?

Reply:
Hmm... not sure! Will look into it, thank you for pointing it out

Reply:
Ah I see, on L142 where filename is constructed it doesn't have the userId, possibly because it hasn't been added to the users collection yet? This isn't causing it to fail so happy for this to merge...